api/integrations/dynatrace/models.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from django.db import models
+
+from environments.models import Environment
+from integrations.common.models import IntegrationsModel
+
+logger = logging.getLogger(__name__)
+
+
+class DynatraceConfiguration(IntegrationsModel):
+    environment = models.OneToOneField(
+        Environment, related_name="dynatrace_config", on_delete=models.CASCADE
+    )
+    entity_selector = models.CharField(max_length=1000, blank=False, null=False)
```

Comment:
I guess this means that we'll expect the FE to send the entity selector string? Have you discussed this with @kyle-ssg ?

Reply:
It's already present in the integrations-data flag config

Reply:
Ok, I'm just concerned that it's not really immediately obvious what data should go in that field (maybe it will be to people using dynatrace) and it seems that it needs to be super specific. Since we're not doing any validation on it on our side, I can foresee some issues with that.

Reply:
Will add to the docs but it will be understood - it's analogous to an AWS ARN.